README.md

```diff
@@ -85,7 +85,7 @@ pipeline step for mabl. This can be written by hand or created via the
 -   continueOnPlanFailure: Continue if your plan/tests fail
 -   environmentId: Selects the environment to run deployments against
     (one of environmentId or applicationId is required)
--   restApiKeyId: The id of the API key secret of the desired deployment workspace -
```

Comment:
ID is the term that Jenkins is using in the user interface.
I would phrase this differently:
restApiKeyId: The id of the API key secret of the desired deployment workspace. The id is the value that is assigned in the Jenkins configuration to the corresponding secret. It is not the secret itself.